lib/s3Uploader.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+var assert = require("assert"),
+    fs = require("fs"),
+    url = require("url");
+
+var Uploader = require("s3-streaming-upload").Uploader;
```

Comment:
Totally an aside: I intend to switch from [s3-streaming-upload](https://www.npmjs.com/package/s3-streaming-upload) to [s3-upload-stream](https://github.com/nathanpeck/s3-upload-stream).

Why? The latter has a better API, doesn't exhibit a bug I've encountered in the former, and isn't written in CoffeeScript.  There's also this: https://github.com/apiaryio/s3-streaming-upload/issues/29#issuecomment-75445871